src/arm/windows/init.c

```diff
@@ -137,7 +134,7 @@ bool get_core_uarch_for_efficiency(
 static bool read_registry(

```

Comment:
While we're at it, make `read_registry` return `wchat_t*` with the text buffer (return `NULL` on error).

Reply:
Thanks for the suggestion, I've added a new commit on top.